src/components/TimelineContainer.jsx

```diff
@@ -0,0 +1,74 @@
+import requestHandler from "../js/requestHandler.js";
+import TweetContainer from "./TweetContainer.jsx";
+
+class TimelineContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            data: [],
+            errorOccurred: false
+        }
+
+        this.updateStatus = (responseStatus, responseData) => {
+            if(responseStatus === 200) {
```

Comment:
can you come up with a design so the rendering code doesn't need to know about the status?